ontoconv/pipelines.py

```diff
@@ -71,11 +91,13 @@ def populate_triplestore(
     with open(yamlfile, encoding="utf8") as f:
         document = yaml.safe_load(f)
```

Comment:
```suggestion
        document = yaml.safe_load(f)
```
I think that using 'document' here is misleading. While we do load a document it is not the fact that it is a document that is the point. We are loading documentation, so I suggest using 'documentation' instead.

Reply:
Agree. Updated